base/ltsockets.dtx

```diff
@@ -766,7 +805,7 @@
 %    does nothing by default other than swallowing its inputs (if any).
 %    \begin{macrocode}
 \cs_new_protected:Npn \socket_new:nn #1 #2 {
-  \str_if_exist:cTF { l_@@_#1_plug_str }
+   \socket_if_exist:nTF {#1}
       {
         \msg_error:nnn { socket } { already-declared } {#1}
       }
```

Comment:
This and several other socket functions use a mixture of 3-space and 2-space indentations. Reformatting can be done after this PR.

Reply:
I you have formatting corrections we might as well add them now,but as you say, we could alternatively do them at a later stage. In any case I would like to get these pulls out of the way this year.

Reply:
Currently comments thus also change suggestions can only be added to the 3 lines before or after a hunk (see https://github.com/github/roadmap/issues/347), thus I cannot suggest needed formatting corrections right in this PR as review comments.

As there are 3-space indentations in other dtx files, indentation can be corrected in a whole, as a solely PR I think.

Reply:
true, so a separate PR seems best (but not really that important -- this comes probably from different editor settings by and will show up again I fear).